### PR TITLE
Machine translation documentation reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: docs/admin/machine.rst [hunk 1]
```diff
@@ -87,10 +88,10 @@ DeepL
 
 .. versionadded:: 2.20
 
-DeepL is paid service providing good machine translation for few languages.
-According to some benchmark it's currently best available service.
```
Comment (comradekingu) [inline docs/admin/machine.rst]: What benchmarks are these?

File: docs/admin/machine.rst [hunk 2]
```diff
@@ -51,8 +50,10 @@ AWS
 Amazon Translate is a neural machine translation service for translating text
 to and from English across a breadth of supported languages.
 
-To enable this service, add ``weblate.machinery.aws.AWSTranslation`` to
-:setting:`MT_SERVICES`, install the `boto3` module and set the settings.
+1. Turn on this service by adding ``weblate.machinery.aws.AWSTranslation`` to
+:setting:`MT_SERVICES`.
+
+2. Install the `boto3` module configure it.
```
Comment (comradekingu) [inline docs/admin/machine.rst]:
```suggestion
2. Install the `boto3` module and configure it.
```

File: docs/admin/machine.rst [hunk 3]
```diff
@@ -51,8 +50,10 @@ AWS
 Amazon Translate is a neural machine translation service for translating text
 to and from English across a breadth of supported languages.
 
-To enable this service, add ``weblate.machinery.aws.AWSTranslation`` to
-:setting:`MT_SERVICES`, install the `boto3` module and set the settings.
+1. Turn on this service by adding ``weblate.machinery.aws.AWSTranslation`` to
+:setting:`MT_SERVICES`.
+
+2. Install the `boto3` module and configure it.
```
Comment (comradekingu) [inline docs/admin/machine.rst]: Is it the module that needs to be configured?

File: docs/admin/machine.rst (hunk 1 above)
Comment (nijel) [reply docs/admin/machine.rst]: This was based on internal benchmarking done by one of Weblate customers. But that's at least two years old, so it probably needs updating and up to date references.

File: docs/admin/machine.rst (hunk 3 above)
Comment (nijel) [reply docs/admin/machine.rst]: No, just Weblate settings.

Comment (comradekingu) [inline docs/admin/machine.rst]:
```suggestion
2. Install the `boto3` module.
3. Configure Weblate.
```

File: docs/admin/machine.rst [hunk 4]
```diff
@@ -87,10 +89,10 @@ DeepL
 
 .. versionadded:: 2.20
 
-DeepL is paid service providing good machine translation for few languages.
-According to some benchmark it's currently best available service.
+DeepL is paid service providing good machine translation for a few languages.
+According to some benchmarks it is currently the best available service.
```
Comment (comradekingu) [inline docs/admin/machine.rst]:
```suggestion
```

**Comment by comradekingu** [comment]
@orangesunny Anything else to be done here?

**Comment by orangesunny** [comment]
@comradekingu Not really, just too many things to review. I will do all the pending reviews soon.

**Comment by orangesunny** [comment]
ready to be published

**Pull request merged**